tests/parental_controls_setup.pm

```diff
@@ -101,9 +101,9 @@ sub run {
     # FIXME: It would be nice to be able to do type-ahead search in this list.
     # See: https://gitlab.gnome.org/GNOME/gnome-control-center/issues/365
     assert_and_click('control_center_users_language', 'left', 10);
-    type_string('English United Kingdom');
```

Comment:
I agree that it would be good for control center to find en_GB given these search terms – after all, these are literally the words displayed in the en_GB row.

Reply:
I fifled https://gitlab.gnome.org/GNOME/gnome-control-center/issues/795.